v20.1/admin-ui-jobs-page.md

```diff
@@ -3,31 +3,78 @@ title: Jobs Page
 toc: true
 ---
 
-The **Jobs** page of the Admin UI provides details about backup/restore jobs, schema changes, [user-created table statistics](create-statistics.html) and [automatic table statistics](cost-based-optimizer.html#table-statistics) jobs, and changefeeds. All users can see their own jobs, and `admin` users can view all jobs performed across all nodes in the cluster. To view these details, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui) and then click **Jobs** on the left-hand navigation bar.
+The **Jobs** page of the Admin UI provides details about long-running tasks your cluster has performed. These can include:
 
+- Schema changes through [`ALTER TABLE`](alter-table.html), [`DROP DATABASE`](drop-database.html), [`DROP TABLE`](drop-table.html), and [`TRUNCATE`](truncate.html).
+- [`IMPORT`](import.html).
+- Enterprise [`BACKUP`](backup.html) and [`RESTORE`](restore.html).
+- [User-created table statistics](create-statistics.html) created for use by the [cost-based optimizer](cost-based-optimizer.html).
+- [Automatic table statistics](cost-based-optimizer.html#table-statistics).
+- [Changefeeds](change-data-capture.html).
 
-## Job details
+{{site.data.alerts.callout_success}}
+All users can see their own jobs, and `admin` users can view all jobs performed across all nodes in the cluster. 
+{{site.data.alerts.end}}
 
-The **Jobs** table displays the ID, description, user, creation time, and status of backup and restore jobs, schema changes, user-created table statistics and automatic table statistics jobs, and changefeeds. To view the job's full description, click the drop-down arrow in the first column.
+To view these details, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui) and click **Jobs** in the left-hand navigation.
 
-<img src="{{ 'images/v20.1/admin_ui_jobs_page_new.png' | relative_url }}" alt="CockroachDB Admin UI Jobs Page" style="border:1px solid #eee;max-width:100%" />
+## Filter jobs
+
+Use the **Status** menu to filter jobs by [job status](#job-status).
+
+Use the **Type** menu to filter jobs by type.
+
+You can toggle between showing the latest 50 jobs or all jobs on the cluster.
+
+{{site.data.alerts.callout_info}}
+Jobs are deleted every 14 days. This interval can be changed via the `jobs.retention_time` [cluster setting](cluster-settings.html). 
+
+If you need a historical record of all jobs you have run, you should log this information externally. 
```

Comment:
Might be worth updating this to be a bit more explicit based on what David said - something like "this interface is not intended to display the canonical record of all jobs that have run.  if you need a historical record ..."  (or something even better of course!)